src/main/java/de/eddyson/tapestry/react/ReactModule.java

```diff
@@ -44,11 +47,17 @@
 import de.eddyson.tapestry.react.services.BabelCompiler;
 import de.eddyson.tapestry.react.services.CJSXCompiler;
 import de.eddyson.tapestry.react.services.NodeBabelCompiler;
+import de.eddyson.tapestry.react.services.nashorn.NashornReactRenderEngine;
+import de.eddyson.tapestry.react.services.nashorn.NashornReactRenderEngineImplementation;
 
 public final class ReactModule {
 
   private final static Logger logger = LoggerFactory.getLogger(ReactModule.class);
 
+  public static void bind(ServiceBinder binder) {
+    binder.bind(NashornReactRenderEngine.class, NashornReactRenderEngineImplementation.class);
```

Comment:
NashornReactRenderEngineImplementation is not thread-safe, it should probably be bound with a per-thread scope.